app/layout.tsx

```diff
@@ -9,7 +9,9 @@ import { Header } from '@/components/header'
 import { Toaster } from '@/components/ui/sonner'
 
 export const metadata = {
-  metadataBase: new URL(`https://${process.env.VERCEL_URL}`),
+  metadataBase: process.env.VERCEL_URL
```

Comment:
I'm wondering if we can just remove this option altogether, as the default behavior seems to be exactly what we want.

https://nextjs.org/docs/app/api-reference/functions/generate-metadata#default-value

> When [VERCEL_URL](https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables#:~:text=.-,VERCEL_URL,-The%20domain%20name) is detected: https://${process.env.VERCEL_URL} otherwise it falls back to http://localhost:${process.env.PORT || 3000}.